app/navigation/root-navigator.tsx

```diff
@@ -242,6 +243,13 @@ export const RootStack = () => {
           title: LL.ReceiveScreen.title(),
         }}
       />
+      <RootNavigator.Screen
+        name="receiveBitcoinFlashcard"
+        component={ReceiveScreenFlashcard}
```

Comment:
I would highly recommend to put name same as screen name. It makes easier to find screen file.
And also, try to put a proper name that matches the feature of the screen: TopupFlashcard